use-cases/upselling.md

```diff
@@ -6,15 +6,23 @@ An upselling integration pulls information about customers, reservations, produc
 
 If the product being posted already exists in Mews, then use [Product order parameters](../operations/services.md#product-order-parameters). If the product does not exist in Mews then use the [Item parameters](../operations/services.md#item-parameters).
 
-To ensure correct reporting, all revenue items posted into Mews must be associated with their correct accounting category by sending the unique identifier in the Item parameters of the Add order request. Information about all the categories configured at each property are requested using [Get all accounting categories](../operations/finance.md#get-all-accounting-categories).
+To ensure correct reporting, all revenue items posted into Mews must be associated with their correct accounting category by sending the unique identifier in the Item parameters of the Add order request. Information about all the categories configured at each property are requested using [Get all accounting categories](../operations/finance.md#get-all-accounting-categories). You can learn more about how accounting categories are created in [this article](https://intercom.help/mews-systems/en/articles/4244319-create-an-accounting-category).
+
+If a product being added to a reservation may require further action from the hotel staff, create reminders and assign responsiblities using [Add task](../operations/enterprises.md#add-task) to ensure no aspect of the guest experience is forgotten. Learn more about how tasks are created and managed in Mews with [this article](https://intercom.help/mews-systems/en/articles/4245631-create-and-manage-tasks).
+
+### Testing your Upselling integration
+
+Ensure you follow our general [guidelines](../guidelines.md) for testing integrations. In addition to this, and specific to upselling integrations
+
+Check whether you are correctly managing the stay product postings by reviewing the Items tab of the reservation you have posted it to or review the product column of the [Reservation report](https://intercom.help/mews-systems/en/articles/4245884-reservation-report) which has `Include products` selected in the Options section. To confirm you are relating any product that is not configured in Mews with the correct accounting category, you can review the Mews [Accounting Report](https://intercom.help/mews-systems/en/articles/4245918-accounting-report). If done correctly, the product you've posted will appear under the relevant accounting category.  
```

Comment:
nit: Include products sounds a bit weird for a report. Please open issue for operations team to rename it to "Display products" which is the format we use on many other reports I believe.